### PR TITLE
feat(PPDSC-2067): audio player v2 live functionality stories and tests

### PR DESCRIPTION
**Pull request by evgenitsn**
PPDSC-2067

**What**

Added Audio Player v2 stories for Live option
Added Audio Player v2 Instrumentation tests
Removed some unnecessary comments

**I have done:**
- [x] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2067-audio-player-v2-live-functionality 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2067-audio-player-v2-live-functionality/storybook

File: src/audio-player-composable/__tests__/audio-player-composable.stories.tsx [hunk 1]
```diff
@@ -25,7 +25,12 @@ import {
   IconFilledLaunch,
   IconFilledReplay5,
   IconFilledForward5,
+  IconFilledGraphicEq,
 } from '../../icons';
+import {Flag} from '../../flag';
+
+const AUDIO_SRC =
+  'https://www.soundhelix.com/examples/mp3/SoundHelix-Song-1.mp3';
```
Comment (mutebg) [inline src/audio-player-composable/__tests__/audio-player-composable.stories.tsx:33]: In the old audio player we use this as URL for Live version, maybe we can use it here as well, so is really live
```
'https://radio.talkradio.co.uk/stream'
```

File: src/audio-player-composable/__tests__/audio-player-composable.test.tsx [hunk 2]
```diff
@@ -241,6 +257,36 @@ describe('Audio Player Composable', () => {
...
+    const playPauseButton = getByTestId('audio-player-play-pause-button');
+
+    fireEvent.canPlay(getByTestId('audio-element'));
+    fireEvent.click(playPauseButton);
+    expect(playPauseButton.getAttribute('aria-label')).toBe('Stop');
+    fireEvent.click(playPauseButton);
+    expect(playPauseButton.getAttribute('aria-label')).toBe('Play');
+  });
+
+  it('should have play or pause label when live', () => {
```
Comment (mutebg) [inline src/audio-player-composable/__tests__/audio-player-composable.test.tsx:275]: I think this is a mistake?
```suggestion
  it('should have play or pause label when recorded', () => {
```

**Review by mutebg — COMMENTED** [review]
Good job @evgenitsn 👏

File: src/audio-player-composable/__tests__/audio-player-composable.stories.tsx (hunk 1 above)
Comment (evgenitsn) [reply src/audio-player-composable/__tests__/audio-player-composable.stories.tsx:33]: Yup, I added this as a second URL which is used for the Live scenarios.

**Review by mutebg — APPROVED** [review]

**Review by Xin00163 — APPROVED** [review]

**Pull request merged**